### ISSUE TITLE
Property:objectType

### ISSUE THREAD

**Issue by mtrekels** [tdwg/cd#64]
|  |  |
| ---- | ---- |
| **Definition** | The physical type of object that is curated as a single specimen |
| **Dimension** | Yes |
| **Existing property** |  |
| **Existing class** |  |
| **Existing property identifier** |  |
| **Format** | Text |
| **Required** |  |
| **Repeatable** |  |
| **Constraints** | Controlled vocabulary |
| **Examples** |  |
| **Notes** |  |

**Comment by Blerina01** [comment]
Might be named as a "taxon", which may be a species, subspecies, genera, family or class.

**Comment by qgroom** [comment]
I think the intention is for this to be the type of object, such as herbarium sheet, bottle, slide, skin, mounted skeleton, box etc. However, it really needs to be made clearer

**Comment by rondlg** [comment]
Hi Quentin, yes that was the intent for this property.

**Comment by mswoodburn** [comment]
Suggestion for definition/existing term from the index spreadsheet: https://www.dublincore.org/specifications/dublin-core/collection-description/collection-application-profile/#colclditemType

**Comment by mswoodburn** [comment]
BioCASE ObjectClassType vocabulary: https://www.biocase.org/doc/Results/WP3/Oldversions/SpyDoc_122/BioCASE_MetaProfile_1.22.shtml#complexType_ObjectClassType_Link01DEB818

**Comment by ekrimmel** [comment]
Is the Dublin Core 'item type' too broad? Since it is similar to and uses the same suggested caovabular as DC 'type' which is [used in DwC](https://dwc.tdwg.org/terms/#dcterms:type) to indicate at a very high level what the object is: PhysicalObject, movingImage, etc.

Also, will this be equivalent to #82 ?

**Comment by qgroom** [comment]
This dimension is quite important for collecting metrics on aggregated collections. I think we need to constrain the users to a strict set of values, otherwise they will not aggregate cleanly

**Comment by qgroom** [comment]
I don't think we need to go to fine detail, as in DwC, because we want to describe collections in common terms.

**Comment by ekrimmel** [comment]
I can see why it's important to provide controlled vocabulary for this property and why this is a crucial property to define well in order to collate metrics across individual collections. I also think this is a tricky property to define and even trickier to provide a controlled vocabulary for that is detailed enough to differentiate object types in interesting ways, but broad enough to apply universally.

In the current definition I think "the physical type of object" and "that is curated as a single specimen" are problematic when combined. **It might be useful to consider whether _objectType_ is designed to fit cataloged objects or physical objects, which are not always equivalent**, e.g. a cataloged object may have multiple physical objects that belong to different collections depending on preparation or research use. You might have a single organism, let's say a bat, for which you prepare a dry study skin (stored in a cabinet), a non-articulated skeleton (stored with the skin), and a DNA sample (stored in a freezer). Those three types of physical objects might be treated as a single cataloged object, i.e. one record in the database, one record on GBIF, citations in the literature for one biological individual that is now one specimen, etc.

Maybe this property just needs to be flexible enough for individual collections to determine their own scope re cataloged objects vs physical objects? In how I'm understanding the current CD model, either way would have implications for other classes: if you count all three bat parts separately then classes like _Storage Location_ and _Object Quantity_ fit more cleanly, but if you count all three bat parts as a single cataloged object then classes like _Taxon Quantity_ and _Digitization Level Assessment_ fit more cleanly.

Arctos has given this a considerable amount of thought. See their rationale for "[what is a cataloged object](http://handbook.arctosdb.org/documentation/catalog#understanding-cataloged-items)" vs. "[what kinds of parts make up a cataloged object](http://handbook.arctosdb.org/documentation/parts.html)" (also see their code table for controlled vocab, which isn't meant to be exhaustive but gives a view into how detailed some collections folks get about object type).

**Comment by mswoodburn** [comment]
As discussed, we've made this a bit more flexible in the definition and removed the controlled vocabulary constraint (although should be recommended to use some form of controlled vocabulary for any CD scheme where using this property as a dimension to group and report on CDs). I've also added in some examples based on NHM Join the Dots data.

**Comment by qgroom** [comment]
> This should generally be used for classifying objects by taxon

Shouldn't this be...

> This should not generally be used for classifying objects by taxon

**Comment by mswoodburn** [comment]
Yes, sorry - very bad word to miss out! I've fixed that in the notes.

**Comment by wouteraddink** [comment]
suggestion to change from objectType to materialAppearance or objectAppearance. See also my comments in https://github.com/tdwg/cd/issues/255. 

controlled terms used in SYNTHESYS+ (the descriptions are for different disciplines so there is some repetition):

Dried assemblage | Not in fluid
Dried - not assembled | Not in fluid, human remains bones,  (not recent)
Fluid preserved |  
Microscope slides |  
Cryopreserved/frozen - 80C |  
Artefacts: climate controlled conditions | Air conditioning/climate controlled units/rooms
Artefacts: non climate controlled conditions | Not air conditioned/climate controlled units/rooms can include mummies
Pressed and dried | Herbarium specimens
Dried | Fruits wood samples, not preserved in fluid.
Fluid preserved | Flowers/fungi in alcohol/formalin/glycerine
Microscopic slides | Microscopic slides
Cryopreserved/frozen 80C | DNA/RNA, tissue
Spore print | Spore print
Dried | Not preserved in fluid
Cryopreserved DNA/RNA | DNA/RNA, tissue
Dried - assembled | Multiple animal parts or entire organism skeletons, stuffed animals
Dried - not assembled | Animal part: tanned skin, egg shell, etc
Fluid preserved | Animals in alcohol/formalin/glycerine
Dried and pinned | Pinned insects
Dried - assembled | Not pinned. Multiple animal parts of entire organism
Dried - not assembled | Animal part. shell, bone, etc.
Fluid preserved | Animals in alcohol/formalin/glycerine
Microscope slides | Microscopic slides
Cryopreserved / frozen -80C | DNA/RNA, tissue
Other |  
Unspecified
Macrofossils (dry preserved) | Handspecimens/slabs/matrix support (i.e. surrounded by original sediment), matrix free (free from original sediment)  - botanical, vertebrates, invertebrates, trace fossils etc.
Mesofossils (dry preserved) | Small fossilised parts of plants such as fruits, leaves, and seeds contained in Jars, Franke cells - i.e. a paper container, the size of a preperation glass with a circular space covered by a lid-covering glass.
Microfossils (dry preserved) | Dry samples, in jars, trays (i.e. not preserved in fluid) etc.
Macrofossils (fluid preserved) | Preserved in a fluid in a  jar, a concealed units.
Messofossils (fluid preserved) | Preserved in a fluid in a  jar, a concealed unit.
Microfossils (fluid preserved) | Preserved in a fluid in a  jar, a concealed unit
Fossils preserved in Amber, natural resin | required to be kept in humidity and light controlled storage units.
Microscope slides | Microscope slides of microfossils, messofossils and macrofossils for either binocular or petrographic microscopes
Oversized fossils | Too large to be fit into standard storage units.
Other | Sieving residue, other microscopic preperations (SEM stubs) etc.
Unspecified
Macro-objects | Handspecimens/hand-held objects/slabs that can be contained in standard units (draws, shelves, cabinets). E.g. rocks, minerals, gems (rough natural form) and ores.
Micro-objects | Can only be handled/observed with the aid of a microscope. Contained in jars
Cut/polished gemstones | High-expense/rare/precious stones that need careful handling and contained in secure units
Microscope slides | Binocular or petrographic microscope slides of rocks, minerals, gems, ore, alloys etc
Cores | Rocks, Ore, Sediments (soil, mud etc.) etc.
Fluids | Hydrocarbons, oils etc.
Oversized objects | Requires extra space because objects are too large for standard units/containers.
Hazardous material/objects | Material or fluids that are hazardous to health - radioactive, toxic etc.
Other | Does not fit into the above subcatagories. e.g. crushed rocks, other microscopic prepared objects (e.g. SEM stubs) etc.
Macro-objects | Handspecimens/hand-held/slabs Meteorites, Moon rock etc
Micro-objects | Can only be handled/observed with the aid of a microscope. contained in Jars, sample bags etc.
Oversized objects | Requires extra space because objects are too large for standard units/containers.
Microscope slides | Thin sections of meteorites etc.

This set is particularly helpful for collection management or image recognition services.

Note: This is NOT the same as BioCASE ObjectClassType vocabulary: https://www.biocase.org/doc/Results/WP3/Oldversions/SpyDoc_122/BioCASE_MetaProfile_1.22.shtml#complexType_ObjectClassType_Link01DEB818 which is more like Material (wood, bark, leaf, gold, ivory etc) although a bit mixed.

**Comment by wouteraddink** [comment]
Note that in Synth+ dashboard development this was called PreservationType which I think was not right, as that should be more something like http://arctos.database.museum/info/ctDocumentation.cfm?table=CTPART_PRESERVATION, see also https://github.com/tdwg/cd/issues/65 (preservationMethod)

**Comment by essvee** [comment]
Updated the property name + examples to reflect that this property/discussion focuses on a lower-level classification than `objectType` which has a new issue assigned: [https://github.com/tdwg/cd/issues/304](https://github.com/tdwg/cd/issues/304).
